pyuvdata/uvdata/mir_meta_data.py

```diff
@@ -1144,6 +1154,8 @@ def get_value(
         idx_arr = self._index_query(use_mask, where, and_where_args, header_key, index)
 
         if isinstance(field_name, (list, set, tuple)):
+            if return_tuples is None:
```

Comment:
Why is the default changed to `None`? Can the kwarg in the function call just be `True`?

Reply:
So I'm using `None` here as a stand-in for what the typical usage is inside of `MirParser`, which is that if multiple fields have been requested, to return tuples (as they're often used for indexing via hashmap), but in cases where single field is requested, just to return the array of values. This is more of a convenience thing inside of `MirParser` rather than anything else -- it just helped to remove about two dozen extra keyword calls inside the code.